### DISCUSSION TITLE
Issue with Deleting Feature Toggle in Unleash

### DISCUSSION THREAD

**Discussion by mai96shani** [Unleash/unleash#7079] [Q&A]
Hi, I am currently using Unleash application version 5.8.1 and Unleash chart version v3.1.3 and  I am experiencing an issue with deleting a feature toggle in Unleash and I was hoping you could provide some assistance.

Here is the process I have been following:

1. Delete toggle configurations using the endpoint "/api/admin/projects/projectName/features/featureToggleName/environments/environment/strategies/strategyId" with the DELETE method and "application/json" as the ContentType.

2. Archive the toggle using the endpoint "/api/admin/projects/project/features/featureToggleName" with the DELETE method and "application/json" as the ContentType.

3. Delete from archive using the endpoint "/api/admin/archive/featureToggleName" with the DELETE method and "application/json" as the ContentType.

However, after the second step, I am encountering the following error from the Unleash server:

[ERROR] /services/scheduler-service.ts - scheduled job failed | id: publishUnannouncedEvents | Error: Invalid number of arguments

I have checked the API documentation and my code to ensure that I am using the correct endpoints and methods. 
I wanted to reach out to you first to see if you could provide any specific guidance or suggestions.

I appreciate any help you can provide 
Thanks, Mai

**Comment by ivarconr** [comment]
This is a confirmed bug in v5.8, and is only producing an error in the log. The error has been fixed and is not present in the most recent version of Unleash (v5.12)
